pyuvsim/tests/test_simsetup.py

```diff
@@ -26,7 +26,8 @@
 herabeam_default = os.path.join(SIM_DATA_PATH, 'HERA_NicCST.uvbeam')
 
 # Five different test configs
-param_filenames = [os.path.join(SIM_DATA_PATH, 'test_config', 'param_10time_10chan_{}.yaml'.format(x)) for x in range(6)]
+param_filenames = [os.path.join(SIM_DATA_PATH, 'test_config', 'param_10time_10chan_{}.yaml'.format(x)) for x in
+                   range(6)]
```

Comment:
This may also be a personal preference thing too but it always felt like `for` was a more natural place to break statements like this over the next line. perhaps I am in the minority though.

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0